views/template.mustache

```diff
@@ -0,0 +1,41 @@
+<html>
```

Comment:
Since this isnt a snippet, it should be deleted. When adding new pages that include a header and footer, we will just copy and paste from an existing file that has a header and footer.